src/components/BuildConfigRevisionsList/BuildConfigRevisionsList.css

```diff
@@ -0,0 +1,3 @@
+.pf-c-tabs.pf-m-vertical .pf-c-tabs__list {
+  max-width: none;
+}
```

Comment:
create new class and apply the property on it, do not address existing CSS classes